diff --git a/haunt/html.scm b/haunt/html.scm
index 5891cd0..b86592b 100644
--- a/haunt/html.scm
+++ b/haunt/html.scm
@@ -112,6 +112,8 @@ list ATTRS and the child nodes in BODY."
     (() *unspecified*)
     (('doctype type)
      (doctype->html type port))
+    (('html-literal html)
+     (display html port))
     (((? symbol? tag) ('@ attrs ...) body ...)
      (element->html tag attrs body port))
     (((? symbol? tag) body ...)
