diff --git a/haunt/html.scm b/haunt/html.scm
index 5891cd0..b86592b 100644
--- a/haunt/html.scm
+++ b/haunt/html.scm
@@ -92,9 +92,9 @@
 list ATTRS and the child nodes in BODY."
   (format port "<~a" tag)
   (for-each (match-lambda
-             ((attr value)
-              (display #\space port)
-              (attribute->html attr value port)))
+              ((attr value)
+               (display #\space port)
+               (attribute->html attr value port)))
             attrs)
   (if (and (null? body) (void-element? tag))
       (display " />" port)
@@ -112,6 +112,8 @@ list ATTRS and the child nodes in BODY."
     (() *unspecified*)
     (('doctype type)
      (doctype->html type port))
+    (('html-literal html)
+     (display html port))
     (((? symbol? tag) ('@ attrs ...) body ...)
      (element->html tag attrs body port))
     (((? symbol? tag) body ...)
@@ -126,5 +128,5 @@ list ATTRS and the child nodes in BODY."
 (define (sxml->html-string sxml)
   "Render SXML as an HTML string."
   (call-with-output-string
-   (lambda (port)
-     (sxml->html sxml port))))
+    (lambda (port)
+      (sxml->html sxml port))))
